pkgdb/src/buildenv/assets/etc/profile.d/0800_cuda.sh

```diff
@@ -19,13 +19,23 @@ activate_cuda(){
   fi
 
   LIB_DIR="$("$_coreutils/bin/realpath" --no-symlinks "$FLOX_ENV/../../lib")"
+  SYSTEM_LIB_DIR=$("$_findutils/bin/find" \
+	  /run/opengl-drivers /lib64 /lib /usr/lib64 /usr/lib /usr/local/lib64 /usr/local/lib \
```

Comment:
The official installer apparently installs to `/opt/nvidia/` [1]


- [1] https://docs.nvidia.com/sdk-manager/sdkm-command-line-install/index.html

Reply:
Fair. I'll add that one.

Reply:
wait,,, that seems to be just the SDK, not the runtime libs. I was looking at: https://download.nvidia.com/XFree86/Linux-x86_64/555.52.04/README/installedcomponents.html